VTEX - Pick and Pack API.json

```diff
@@ -0,0 +1,3856 @@
+                    }
+                },
+                "example": {
+                    "id": "d39e8dd7-0ab9-48c8-b16b-3e46933e113a",
+                    "serviceId": "550693447769717380942093217914",
+                    "orderId": "1506500515925-01",
+                    "carrierId": "627eb3b93a19fdc0cf5fe087",
+                    "status": "DELIVERED",
+                    "carrierName": "Lastmile",
+                    "paymentMethod": "ONLINE",
+                    "tags": [],
+                    "deliveredDate": "2025-01-31T19:42:04.953Z",
+                    "customerId": "-",
+                    "customerAddressId": "Bello - Medellin",
+                    "source": "Vancouver-CAN",
+                    "destination": "Medellín-COL",
+                    "courierId": "1977157",
+                    "courierName": "adrianprod adrianprod",
+                    "seller": "lastmile",
+                    "packagesQuantity": 1,
+                    "itemsQuantity": 6,
+                    "serviceType": "PICKUP_DELIVERY",
+                    "creationDate": "2025-01-28T19:56:48.033Z"
+                }
+            },
+            "400OrderIDisrequired1": {
+                "required": ["type", "code", "message", "date", "requestId"],
+                "type": "object",
+                "properties": {
+                    "type": {
+                        "type": "string",
+                        "description": "Type of error."
+                    },
+                    "code": {
+                        "type": "string",
+                        "nullable": true,
+                        "description": "Error code."
+                    },
+                    "message": {
+                        "type": "string",
+                        "description": "Error message."
+                    },
+                    "date": {
+                        "type": "string",
+                        "description": "Date and time when the error occurred."
+                    },
+                    "requestId": {
+                        "type": "string",
+                        "description": "Request ID."
+                    }
+                },
+                "example": {
+                    "type": "business",
+                    "code": null,
+                    "message": "Query string parameter {orderId} is required",
+                    "date": "2025-02-13T12:14:18.795Z",
+                    "requestId": "1bb652b1-dc75-4f0c-9c0a-3cd021eaa1ea"
+                }
+            }
+        },
+        "securitySchemes": {
+            "Authorization": {
+                "type": "http",
+                "scheme": "bearer",
+                "bearerFormat": "JWT",
+                "description": "Authentication JWT creted by the [Generate JWT token](). JSON Web Token (JWT) is an open standard (RFC 7519) that defines a compact and self-contained way for securely transmitting information as a JSON object between parties."

```

Comment:
The auth endpoint link is missing.

Also, I feel like it is not necessary to explain what is a JWT since it is neither VTEX domain nor necessary to use the endpoint.